db/migrate/20230617001050_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: { to_table: :users }
+      t.string :title
+      t.text :text
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+      t.timestamps
+    end
+    add_index :posts, :author_id
```

Comment:
- [ ] I think you don't need this libe since you already have it in line 4 so I recommend to remove it 🙏🏻🔥🤩 and then you can run your migrations again to see the schema 🔥

Reply:
Thank you for the review